modules/unsupported/csv/src/main/java/org/geotools/data/csv/CSVFeatureStore.java

```diff
@@ -0,0 +1,141 @@
+/* GeoTools - The Open Source Java GIS Toolkit
+ * http://geotools.org
+ *
+ * (C) 2010-2014, Open Source Geospatial Foundation (OSGeo)
```

Comment:
Looked back at this and I checked an [older snapshot of geoserver](https://github.com/travislbrundage/geoserver/tree/db287497cfdfbe7cff827c3c400be40113462461/src/extension/importer/core/src/main/java/org/geoserver/importer/csv), and this file was never present there.

What happened was the original gt-csv module was made for geotools and had this file.
Geoserver forked it and got rid of the write functionality, including removing this file.
It got backported into geotools again.
Now I am updating it to have write functionality again, this time with strategy objects, and now the code is here. So, I believe it is fine.